src/main/java/service/TransactionServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package service;
+
+import dao.FruitTransactionDao;
+import java.util.HashMap;
+import java.util.Map;
+import model.FruitTransaction;
+import operation.OperationStrategy;
+
+public class TransactionServiceImpl implements TransactionService {
+    private final OperationStrategy operationStrategy;
+    private final FruitTransactionDao fruitTransactionDao;
+
+    public TransactionServiceImpl(OperationStrategy operationStrategy,
+                                  FruitTransactionDao fruitTransactionDao) {
+        this.operationStrategy = operationStrategy;
+        this.fruitTransactionDao = fruitTransactionDao;
+    }
+
+    @Override
+    public Map<String,Integer> countsFruitsAfterWorkDay() {
+        Map<String,Integer> updatedFruitsStockMap = new HashMap<>();

```

Comment:
lets put these fruits not in the local Map, but in some Storage with Fruits and their quantities